core/src/test/java/com/graphhopper/reader/osm/conditional/ConditionalValueParser_ConditionState_DiffblueTest.java

```diff
@@ -0,0 +1,69 @@
+/**
+ *
+ *
+ * Copyright 2018 Diffblue Limited
+ *
+ * Diffblue Limited licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.graphhopper.reader.osm.conditional;
+
+import com.graphhopper.reader.osm.conditional.ConditionalValueParser.ConditionState;
+import org.junit.Assert;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.ExpectedException;
+
+public class ConditionalValueParser_ConditionState_DiffblueTest {
+
+  @Rule public ExpectedException thrown = ExpectedException.none();
+
+  /* testedClasses: ConditionalValueParser_ConditionState */
+
+  /*
+   * Test generated by Diffblue Deeptest.
+   * This test case covers:
+   *  - conditional line 33 branch to line 34
+   */
+
+  @Test
+  public void isCheckPassedOutputIllegalStateException() {
+
+    // Arrange
+    final ConditionState objectUnderTest = ConditionState.INVALID;
+
+    // Act
+    thrown.expect(IllegalStateException.class);
+    objectUnderTest.isCheckPassed();
+
+    // Method is not expected to return due to exception thrown
+  }
+
+  /*
+   * Test generated by Diffblue Deeptest.
+   * This test case covers:
+   *  - conditional line 15 branch to line 15
+   */
+
+  @Test
+  public void valueOfInputNotNullOutputIllegalArgumentException() {
```

Comment:
I can't find the source for this function?

Reply:
@paulkennethkent Here it is: https://github.com/Diffblue-benchmarks/graphhopper/blob/master/core/src/main/java/com/graphhopper/reader/osm/conditional/ConditionalValueParser.java#L15

Reply:
@paulkennethkent Actually, i'm going to remove this one. Trace view breaks, so its obvious something is messed up.